### PR TITLE
ecs: add v8 alias to v1 implementation

### PR DESCRIPTION
**Pull request by yaauie**

### CODE REVIEW

**Review by kares — APPROVED** [review]
:cherries:

**Review by kares — APPROVED** [review]
:ice_cream:

**Review by kares — CHANGES_REQUESTED** [review]
sorry wrong tab on previous :heavy_check_mark: 

since we need to pass [`ecs_compatibility` to the grok filter](https://github.com/logstash-plugins/logstash-input-syslog/blob/v3.5.0/lib/logstash/inputs/syslog.rb#L107) we need to make sure logstash-filter-grok is updated first and it's version requirement is updated afterwards with v8 support, sample failing test:

https://app.travis-ci.com/github/logstash-plugins/logstash-input-syslog/jobs/527490671#L516-L517

**Comment by yaauie** [comment]
upstream grok PR https://github.com/logstash-plugins/logstash-filter-grok/pull/175

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+## 3.6.0
+  - Add support for ECS v8 as alias to v1 implementation
```
Comment (yaauie) [inline CHANGELOG.md:2]:
```suggestion
  - Add support for ECS v8 as alias to v1 implementation [#68](https://github.com/logstash-plugins/logstash-input-syslog/pull/68)
```

**Review by kares — APPROVED** [review]
LGTM, the failing CI against 8.0-SNAPSHOT is unrelated

(an issue with timestamp formatting due changes in LS 8.0 e.g. `expected: "2021-10-26T20:19:25.000Z" got: "2021-10-26T20:19:25Z"`)

**Comment by yaauie** [comment]
@kares I resolved the failing spec on LS8 with a8c0036 which uses a helper introduced in devutils 2.3 to make assertions about timestamps that are not bound to the serialization length.

**Review by kares — APPROVED** [review]
LGTM, :green_heart: CI

**Pull request merged**